configure.ac

```diff
@@ -267,29 +255,96 @@ dnl --- enable_pkey
 enable_pkey=no
 
 case $target in
-     *s390x*)
-	  enable_pkey=yes
-	  ;;
-     *s390*)
-	  CFLAGS="$CFLAGS -m31 -DNO_PKEY"
-	  ;;
-     *ppc64* | *x86_64*)
-	  CFLAGS="$CFLAGS -DNO_PKEY"
-	  ;;
-     *ppc* | i*86*)
-	  CFLAGS="$CFLAGS -m32 -DNO_PKEY"
-	  ;;
-     *)
-	  CFLAGS="$CFLAGS -DNO_PKEY"
-	  ;;
+	*s390x*)
+		enable_pkey=yes
+	;;
+	*s390*)
+		CFLAGS="$CFLAGS -m31"
+		AC_DEFINE([NO_PKEY])
+	;;
+	*ppc64* | *x86_64*)
+		AC_DEFINE([NO_PKEY])
+	;;
+	*ppc* | i*86*)
+	  CFLAGS="$CFLAGS -m32"
+		AC_DEFINE([NO_PKEY])
+	;;
+	*)
+		AC_DEFINE([NO_PKEY])
+	;;
 esac
 AM_CONDITIONAL([NO_PKEY], [test "x$enable_pkey" != "xyes"])
 
+dnl Check and save whether the platform is AIX
+dnl Using the pthread library requires the _THREAD_SAFE macro to be set
+build_aix=no
+case "${host_os}" in
+	aix*)
+		build_aix=yes
+		AC_DEFINE([_ALL_SOURCE])
+		AC_DEFINE([_THREAD_SAFE])
+		with_systemd=no
+		with_libica=no
+		enable_ccatok=yes
+		enable_swtok=yes
+		enable_tpmtok=no
+		enable_icsftok=no
+		enable_ep11tok=no
+		echo "Disabling migration utilities on AIX: pkcscca pkcstok_migrate"
+		enable_pkcscca=no
+		enable_pkcstok_migrate=no
+		echo "Disabling the following unsupported features on AIX: systemd libica pmtok icsftok ep11tok"
+		pkcsslotd_user=slotd
+		echo "Shortening username to $pkcsslotd_user due to platform restrictions"
+		localstatedir='$(DESTDIR)/var'
+		sysconfdir='$(DESTDIR)/etc'
+		function parse_map {
+			grep '^        ' $1 | tr -d ';' | tr -d ' ' > "$1".sym
+		}
+		parse_map opencryptoki.map
+		parse_map opencryptoki_tok.map
+		;;
+esac
+
+dnl Define custom variables
+lockdir=$localstatedir/lock/opencryptoki
+logdir=$localstatedir/log/opencryptoki
+AC_SUBST(lockdir)
+AC_SUBST(logdir)
+AM_CONDITIONAL([AIX], [test "x$build_aix" = "xyes" ])
+
+dnl define _GNU_SOURCE on non-AIX platforms
+if test "x$build_aix" = "xno"; then
+	AC_DEFINE([_GNU_SOURCE])
+fi
+
 dnl --- enable_debug
 if test "x$enable_debug" = "xyes"; then
-	CFLAGS="$CFLAGS -gdwarf-2 -g3 -O0 -DDEBUG"
+	if test "x$build_aix" = "xyes"; then
+		CFLAGS="$CFLAGS -g3 -O0"
+	else
+		CFLAGS="$CFLAGS -gdwarf-2 -g3 -O0"
+	fi
+	AC_DEFINE([DEBUG])
+else
+	CFLAGS="$CFLAGS -O2 -g3"
 fi
 
+if test "x$build_aix" = "xno"; then
+	LIBCAP_LIBS=
+	AC_CHECK_LIB([cap], [cap_get_proc], [LIBCAP_LIBS="-lcap"],
+			[AC_MSG_ERROR(['libcap' library is missing on your system. Please install 'libcap-devel'.])])
+	AC_SUBST([LIBCAP_LIBS])
+fi
+
+dnl --- cca and soft tokens need lber, icsf needs only ldap
+OPENLDAP_LIBS=
+AC_CHECK_HEADERS([lber.h ldap.h],
+		[OPENLDAP_LIBS="-llber -lldap"],
+		[AC_MSG_ERROR([lber.h and ldap.h are missing. Please install
+									'openldap-devel'.])])
```

Comment:
Is this intentionally indented with so many blanks/tabs ? I would say the indention was correct before.....